adapters/adformOpenRTB/adformOpenRTB.go

```diff
@@ -0,0 +1,106 @@
+package adformOpenRTB
```

Comment:
Package names in Go are preferred to be lower case. Please consider naming your package and folder adformopenrtb.